blog/2015/11/swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,436 @@
+  <li>Jon Pipitone</li>
+  <li>John Moreau</li>
+  <li>Gayathri Swaminathan</li>
+  <li>Nichole Bennett</li>
+  <li>Tracy Teal</li>
+  <li>Raniere Silva</li>
+  <li>Sarah Stevens</li>
+  <li>Lex Nederbragt</li>
+  <li>Kate Hertweck</li>
+  <li>Greg Wilson</li>
+  <li>Becca Tarvin</li>
+  <li>Azalee Bostroem</li>
+</ul>
+
+Hopefully I got everyone!
+
+<h2>Motivation</h2>
+Last year I <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002202.html'>asked</a> the <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/'>discuss list</a>
+if anyone had any experience with turing the <a href='http://software-carpentry.org/lessons.html'>software-carpentry lessons</a> into a university course.
+Bill Mills <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002203.html'>mentioned</a> that two main hurdles revolve around politics and curriculum control.
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002204.html'>Ethan White</a>
+and <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002211.html'>Michael Jackson</a>went on to describe their own experiences with settin up a course.
+I want to believe the push for code sharing and reproducibility article from
+<a href='http://www.nature.com/news/interactive-notebooks-sharing-the-code-1.16261'>Nature</a> and submission guidelines for
+<a href='http://journals.plos.org/plosone/s/materials-and-software-sharing'>PLOS ONE</a>
+have addressed <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002205.html'>Greg's</a> initial observations that universites find that lab skills course is worth graduate credit.  The conversation contines
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002207.html'>here</a>,
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002210.html'>here</a>, and
+a blog post by Damien Irving <a href='http://software-carpentry.org/blog/2014/02/university-course.html'>here</a>.
+
+The Software Carpentry + Data Carpentry Instructor & Helper Retreat seemed like a good means to revisit the discussion.
+
+<h2>Who are we?</h2>
+<a href='https://twitter.com/chendaniely'>Daniel Chen</a> is a doctoral candidate in
+<a href='http://gbcb.vbi.vt.edu/gbcb/about_program'>Genetrics, Bioinformatics and Computational Biology (GBCB)</a>
+at Virginia Tech and currently working in the
+<a href='http://vbi.vt.edu/sdal'>Social and Decision Analytics Laboratory</a>
+under the <a href='http://vbi.vt.edu'>Virginia Bioinformatics Institute</a>.
+I received my MPH at the
+<a href='http://www.mailman.columbia.edu'>Mailman School of Public Health</a> in Epidemiology.
+
+Tiffany Timbers is ...
```

Comment:
@wking Good idea, I didn't think of doing that.

I guess @jennybc and @ttimbers I'll just link your swc team page link here.
Unless you want to add extra information that is not on the team page that you would think is relevant to this post